src/lightning/pytorch/demos/boring_classes.py

```diff
@@ -19,6 +19,7 @@
 from torch import Tensor
```

Comment:
we said we would exclude examples

Reply:
you did later qualify after I started that you didn't want overrides in examples. guess I missed this `demos` directory when I removed them from 14 other files here: https://github.com/Lightning-AI/pytorch-lightning/pull/18866/commits/404514cd2bc47ebbf6970b77b53b920d1be8b2f4. sorry about that!

Reply:
sorry, I meant 25 files.